common/test/acceptance/pages/studio/library.py

```diff
@@ -252,12 +252,19 @@ def get_metadata_input(self, metadata_key):
         return element
 
 
+@js_defined('window.LibraryContentAuthorView')
 class StudioLibraryContainerXBlockWrapper(XBlockWrapper):
     """
     Wraps :class:`.container.XBlockWrapper` for use with LibraryContent blocks
     """
     url = None
 
+    def is_browser_on_page(self):
+        """
+        Returns true iff the library content area has been loaded
```

Comment:
No, that is intentional. "Iff" means "If and only if" - it's a stronger form of "if" ( http://en.wikipedia.org/wiki/If_and_only_if )